roles/contiv_network/files/v2plugin.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -x
+sleep 2
+ID="$(docker plugin ls  | awk '/contiv/ {print $2}')"
+sudo docker plugin enable $ID
```

Comment:
since it will be run by systemd, no sudo required

Reply:
Good catch.  If there was an issue with writing to the Docker socket which necessitated sudo, line 4 would fail anyways.

Reply:
Ok